packages/core/nuxt-module/lib/module.js

```diff
@@ -43,6 +44,13 @@ module.exports = function VueStorefrontNuxtModule (moduleOptions) {
   this.addPlugin(path.resolve(__dirname, 'plugins/ssr.js'))
   log.success('Installed VSF SSR plugin');
 
+  this.addPlugin({
```

Comment:
shouldn't we run this only if custom logger is provided?

Reply:
nope, because you need to run logger configuration for example for verbosity etc. but.... I  think you mean to load this plugin only if section logger in the config is provided and in that case - yes, will do

Reply:
yup, exactly. default logger should be applied on the library level ;)